src/upload/application/views/install/in_database_view.php

```diff
@@ -16,7 +16,7 @@
                 <label>{ins_user_title}</label>
                 <input type="text" name="user" placeholder="{ins_ex_tag} root" value="{v_user}">
                 <label>{ins_password_title}</label>
-                <input type="text" name="password" value="" autocomplete="off">
+                <input type="password" name="password" value="" autocomplete="off">
```

Comment:
@duhow this is intended. Since we do not have password verification field, we won't mask this